assets/app/views/create/nextSteps.html

```diff
@@ -0,0 +1,102 @@
+<div ng-controller="ProjectController">
+  <div ng-controller="CreateFinishedController" class="container next-steps">
+    <ol class="breadcrumb">
```

Comment:
I've added a breadcrumbs directive in my PR.  Add a comment here to come back and update it to use that.